src/templates/static/get_started/index.jsx

```diff
@@ -42,6 +43,70 @@ const CTA = () => (
     </div>
 );
 
+const VolatilityIndicesTable = () => {
+    const active_symbols = [
+        {
+            key             : 'r10',
+            label           : it.L('Volatility Indices 10'),
+            high_close_value: 86.85,
```

Comment:
as I understand it needs to be rounded to the nearest integer, i.e. `87`

Reply:
ah, I see that you round it later. up to you, I don't see a point.

Reply:
I was not sure if these values are going to change later or not. If by any chance we make this table dynamic, Math.round will help us later.